crates/holochain/tests/test_utils/mod.rs

```diff
@@ -370,16 +369,15 @@ async fn check_timeout_named<T>(
     name: &'static str,
     response: impl Future<Output = std::io::Result<T>>,
     timeout_millis: u64,
-) -> T {
-    // FIXME(stefan): remove this multiplier once it's faster on self-hosted CI
-    let timeout_millis = timeout_millis * 4;
-    match tokio::time::timeout(std::time::Duration::from_millis(timeout_millis), response).await {
-        Ok(response) => response.unwrap(),
-        Err(e) => {
-            panic!(
```

Comment:
When there are multiple call paths to this that use `<unnamed>`, it's hard to work out which one. Same as we've done other places, bubble the error up so you can see where the test got to